### DISCUSSION TITLE
Support GraphQL Subscriptions

### DISCUSSION THREAD

**Discussion by VinayaSathyanarayana** [keystonejs/keystone#5355] [Feature requests]
Please support GraphQL Subscriptions.
Supported by Apollo Client: https://www.apollographql.com/docs/react/data/subscriptions/

**Comment by bladey** [comment]
Hi @VinayaSathyanarayana, thanks for the suggestion, see here for an update on subscriptions - https://keystonejs.slack.com/archives/C01STDMEW3S/p1624618723155700

**Comment by VinayaSathyanarayana** [comment]
References:
https://www.npmjs.com/package/graphql-ws
https://www.npmjs.com/package/graphql-subscriptions
